VideoWeb/VideoWeb/Controllers/ConferencesController.cs

```diff
@@ -27,12 +29,30 @@ public class ConferencesController : Controller
         private readonly IVideoApiClient _videoApiClient;
         private readonly ILogger<ConferencesController> _logger;
         private readonly IConferenceCache _conferenceCache;
+        private readonly IMapTo<ConferenceForJudgeResponse, JudgeConference> _conferenceForJudgeResponseMapper;
+        private readonly IMapTo<ConferenceForIndividualResponse, IndividualConference> _conferenceForIndividualResponseMapper;
+        private readonly IMapTo<ConferenceForVhOfficerResponse, ConferenceForAdminResponse> _conferenceForVhOfficerResponseMapper;
+        private readonly IMapTo<ConferenceResponseVho, ConferenceDetailsResponse> _conferenceResponseVhoMapper;
+        private readonly IMapTo<ConferenceResponse, ConferenceDetailsResponse> _conferenceResponseMapper;
 
-        public ConferencesController(IVideoApiClient videoApiClient, ILogger<ConferencesController> logger, IConferenceCache conferenceCache)
+        public ConferencesController(
+            IVideoApiClient videoApiClient,
+            ILogger<ConferencesController> logger,
+            IConferenceCache conferenceCache,
+            IMapTo<ConferenceForJudgeResponse, JudgeConference> conferenceForJudgeResponseMapper,
```

Comment:
Any chance we can avoid listing so many mappers in the ctor but rather a single IMapTo factory

Reply:
changed